azure-pipelines.yml

```diff
@@ -267,7 +267,10 @@ stages:
       - group: Publish-Build-Assets
     jobs:
     - template: /eng/common/templates/job/job.yml
-      parameters: 
+      parameters:
+        pool:
+          name: $(PoolProvider) # This is a queue-time parameter; Public default is NetCorePublic-Pool; Internal default is NetCoreInternal-Pool
```

Comment:
These aren't the correct names for the pool provider, if you want to fix that....
It's NetCore1ESPool-Internal / NetCore-Public, these are the ancient "DIY Pool provider' names.